docs/vagrant.md

```diff
@@ -37,15 +36,14 @@ You can use `kubectl` from now on. Test it by executing `kubectl get nodes`
 
 ### Multi node PKE
 
-Start a machine with the following command: `vagrant up node1 node2`
+Start a machine with the following command: `vagrant up centos1 centos2`
```

Comment:
this should be changed in the pipeline quickstart guide as well

Reply:
Once this got merged, sure.